src/Mortgage/Mortgage.router.js

```diff
@@ -0,0 +1,67 @@
+import { server, utils } from 'decentraland-commons'
+
+import { Mortgage } from './Mortgage.model'
+import { Parcel } from '../Parcel'
+import { blacklist } from '../lib'
+
+export class MortgageRouter {
+  constructor(app) {
+    this.app = app
+  }
+
+  mount() {
+    /**
+     * Get parcels with mortgages by borrower
+     * @param  {string} address
+     * @return {array<Parcel>}
+     */
+    this.app.get(
+      '/api/parcels/:address/mortgages',
+      server.handleRequest(this.getMortgagedParcelsByBorrower)
+    )
+
+    /**
+     * Get mortgages by borrower
+     * @param  {string} address
+     * @return {array<Mortgage>}
+     */
+    this.app.get(
+      '/api/addresses/:address/mortgages',
+      server.handleRequest(this.getMortgagesByBorrower)
+    )
+
+    /**
+     * Get mortgages by coordinates
+     * @param  {string} x
+     * @param  {string} y
+     * @return {array<Mortgage>}
+     */
+    this.app.get(
+      '/api/parcels/:x/:y/mortgages',
+      server.handleRequest(this.getActiveMortgagesInCoordinate)
+    )
+  }
+
+  async getMortgagedParcelsByBorrower(req) {
+    const borrower = server.extractFromReq(req, 'address')
+    const parcels = await Parcel.findWithLastActiveMortgageByBorrower(borrower)
+    return utils.mapOmit(parcels, blacklist.parcel)
+  }
+
+  async getMortgagesByBorrower(req) {
+    const borrower = server.extractFromReq(req, 'address')
+    return await Mortgage.findActivesByBorrower(borrower)
+  }
+
+  async getActiveMortgagesInCoordinate(req) {
+    const x = server.extractFromReq(req, 'x')
+    const y = server.extractFromReq(req, 'y')
+    const status = server.extractFromReq(req, 'status')
+
+    let mortgages = []
+    if (status === 'active') {
+      mortgages = await Mortgage.findActivesInCoordinate(Parcel.buildId(x, y))
+    }
+    return mortgages
```

Comment:
this is a bit weird to me, if we're only supporting active, why does the endpoint have an `status` param?

Reply:
Well, I hace the same feeling when doing it.

Yes, The endpoint handler should be named as `getMortgagesInCoordinate`

About `status`, although`active` is not a valid status for mortgages, will let us use the request without `status` to retrieve all mortgages.

Reply:
> About status, althoughactive is not a valid status for mortgages, will let us use the request without status to retrieve all mortgages. E.g: if we want to get a mortgages history. Dashboard data, etc

True! but we can also accomplish that by keeping the `active` abstraction in the front-end and supporting an array of status on this request. Which will work for future references and it's easier to make public (we won't have to explain what active means)

Reply:
Im not against top level abstractions which help users memo ways to use api. Said that, Im ok to support status array, but also the other way.

Anyways, so far lets go with array support only